src/components/ui/UIButton/UIButton.tsx

```diff
@@ -0,0 +1,151 @@
+import {
+  Interpolation,
+  Theme,
+  Button,
+  ThemeProvider,
+  createTheme,
+} from '@mui/material';
+import React from 'react';
+
+import { UIButtonProps } from './UiButtonTypes';
+
+const defaultProps: UIButtonProps = {
+  variant: 'contained',
+  size: 'small',
+  disabled: false,
+  disableElevation: false,
+  disableFocusRipple: false,
+  disableRipple: false,
+  fullWidth: false,
+  href: '',
+  children: '',
+};
+
+export const repeatStyles: Interpolation<{ theme: Theme }> = {
+  textTransform: 'none',
+  textDecoration: 'none',
+  fontSize: '0.938rem',
+  fontWeight: '500',
+  lineHeight: '1.125',
+};
+
+const theme = createTheme({
```

Comment:
need move this theme to another file, for example Theme.ts, this part make component more complicated